packages/backend/src/modules/user/user.service.ts

```diff
@@ -27,22 +34,28 @@ export default class UserService {
   public async login(email: string, password: string, device: string): Promise<ITokens> {
     const user = await this.userModel.findOne({ email });
 
+    if ((await this.isEmailNotConfirmed(email))) {
+      await this.sendEmailConfirmationCode(email);
+      throw new AuthenticationError('We have sent a confirmation to you. Confirm your email address, please.');
```

Comment:
Please verify your email by using a confirmation code we have sent to your email.